blockchain/monitor/event_setup.go

```diff
@@ -13,43 +13,72 @@ import (
 func SetupEventMap(em *objects.EventMap, cdb *db.Database, adminHandler interfaces.AdminHandler, depositHandler interfaces.DepositHandler) error {
```

Comment:
Do not hardcode the event hashes as this will result in silent failure when an event is updated.

Reply:
parse from bindings